recipe/meta.yaml

```
@@ -8,8 +8,10 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: http://www.openssl.org/source/{{ name }}-{{ version }}.tar.gz
-  sha256: 5c557b023230413dfb0756f3137a13e6d726838ccd1430888ad15bfb2b43ea4b
-
+  sha256: f6fb3079ad15076154eda9413fed42877d668e7069d9b87396d0804fdb3f4c90
+  patches:
+    - 0001-Don-t-use-USE_BCRYPTGENRANDOM-for-VS-2019.patch
```

```suggestion
    - 0001-Don-t-use-USE_BCRYPTGENRANDOM-for-VS-2019.patch
```

Isn't the patch for older VS?  this patch name sort of implies that it's used for brand new VS to me.  The incompatible thing is the MS SDK.  _MSC_VER is probably a good enough proxy here.  That 1900 corresponds to VS 2015 (yeah, the versioning here is really, really hard)